SmartDeviceLink_Example/Classes/ProxyManager.m

```diff
@@ -145,15 +142,15 @@ - (void)setupPermissionsCallbacks {
     NSArray *rpcGroup =@[@"AddCommand", @"PerformInteraction"];
     SDLPermissionGroupStatus commandPICSStatus = [self.sdlManager.permissionManager groupStatusOfRPCs:rpcGroup];
     NSDictionary *commandPICSStatusDict = [self.sdlManager.permissionManager statusOfRPCs:rpcGroup];
-    NSLog(@"Command / PICS status: %@, dict: %@", @(commandPICSStatus), commandPICSStatusDict);
+    SDLLogD(@"Command / PICS status: %@, dict: %@", @(commandPICSStatus), commandPICSStatusDict);
 
     // This will set up a long-term observer for the RPC group and will tell us when the status of any specified RPC changes (due to the `SDLPermissionGroupTypeAny`) option.
     [self.sdlManager.permissionManager addObserverForRPCs:rpcGroup groupType:SDLPermissionGroupTypeAny withHandler:^(NSDictionary<SDLPermissionRPCName, NSNumber<SDLBool> *> * _Nonnull change, SDLPermissionGroupStatus status) {
-        NSLog(@"Command / PICS changed permission to status: %@, dict: %@", @(status), change);
+        SDLLogD(@"Command / PICS changed permission to status: %@, dict: %@", @(status), change);
     }];
 }
 
-+ (SDLLifecycleConfiguration *)setLifecycleConfigurationPropertiesOnConfiguration:(SDLLifecycleConfiguration *)config {
++ (SDLLifecycleConfiguration *)sdlex_setLifecycleConfigurationPropertiesOnConfiguration:(SDLLifecycleConfiguration *)config {
```

Comment:
It would be easier to understand the function's purpose if the method name was `sdl_ex_setLifecycleConfigurationPropertiesOnConfiguration`  or `sdlExample_setLifecycleConfigurationPropertiesOnConfiguration` instead of `sdlex_setLifecycleConfigurationPropertiesOnConfiguration`